_shared/project/requirements/functests.in

```diff
@@ -6,6 +6,7 @@ pytest
 factory-boy
 pytest-factoryboy
 h-matchers
+h-testkit
```

Comment:
This will add h-testkit to all projects dependencies indiscriminately, even if the project doesn't currently need any of h-testkit's current fixtures. I think this is probably simplest and most convenient. This way if any project does want to use something from h-testkit in future, it's already there